requirements.txt

```diff
@@ -18,6 +18,9 @@ gdown
 gensim==3.8.3
 sklearn
 gdown
+readability-lxml
+bs4
```

Comment:
I found (their pypi page)[https://pypi.org/project/bs4/#description] and it seems that this is a name holder for the (actual package)[https://pypi.org/project/beautifulsoup4/]?